app/controllers/idv/personal_key_controller.rb

```diff
@@ -38,6 +45,25 @@ def update
       redirect_to next_step
     end
 
+    def self.step_info
+      Idv::StepInfo.new(
+        key: :personal_key,
+        controller: self,
+        action: :show,
+        next_steps: [FlowPolicy::FINAL],
+        preconditions: ->(idv_session:, user:) do
+          return false unless idv_session.address_confirmed? || idv_session.phone_confirmed?
+
+          return false unless user.active_or_pending_profile
+
+          !idv_session.personal_key_acknowledged
```

Comment:
Do we need to prevent the user from seeing this page if they already acknowledged their personal key? That sounds like a back button prevention check.